pkg/resources/task.go

```diff
@@ -182,7 +182,7 @@ func ReadTask(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	if err := d.Set("enabled", task.IsStarted()); err != nil {
+	if err := d.Set("enabled", task.State == sdk.TaskStateStarted); err != nil {
```

Comment:
do we want to set enabled if this is not set in the config? (maybe it is already a topic for the next change but had to ask; we are going away from the defaults for such attributes; it's also similar in a way to the warehouse dilemma we had for the state that can be changes externally - let's at least discuss it)

Reply:
Don't remember where, but I wrote somewhere that it's very hard or impossible to act on the default `enabled` value (not set in the config). That's why I made it required, so certain operations are easier because we know what state are we aiming with. This was truly painful during the update where suspend/resume logic is messy because of the Snowflake limitations. Maybe it will be possible to turn it into a tri-valued bool, but for now, for simplicity's sake, I would stay with the required regular bool.